ebd/ebuild.bash

```diff
@@ -348,6 +348,7 @@ __run_ebuild_phase() {
 	source "${PKGCORE_EBD_PATH}"/.generated/libs/${EAPI}/$1 \
 		|| die "failed sourcing EAPI ${EAPI} $1 phase scope lib"
 
+	cd "${PKGCORE_EMPTYDIR}" || die "unable to cd into ${PKGCORE_EMPTYDIR}"
```

Comment:
Isn't this supposed to only be done for pkg_* phases? This runs it for every phase.

Reply:
Do you see any harm in doing it unconditionally, then having phases with different definition change workdir? It seemed cleaner to me this way.

Reply:
> Do you see any harm in doing it unconditionally, then having phases with different definition change workdir? It seemed cleaner to me this way.

Not off the top of my head... this would be one of those merge and see if it breaks the few people who use pkgcore for build/merging kind of commits that I was alluding to. :laughing: